src/likelihoodratiotest.jl

```diff
@@ -215,3 +255,49 @@ function StatsModels.isnested(m1::MixedModel, m2::MixedModel; atol::Real=0.0)
 
     true
 end
+
+
+"""
+    isnested(m1::LinearModel, m2::LinearMixedModel; atol::Real=0.0)
+    isnested(m1::TableRegressioLinearModel{LinearModel}, m2::LinearMixedModel; atol::Real=0.0)
+    isnested(m1::GeneralizedLinearModel, m2::GeneralizedLinearMixedModel; atol::Real=0.0)
+    isnested(m1::TableRegressioLinearModel{GeneralizedLinearModel}, m2::GeneralizedLinearMixedModel; atol::Real=0.0)
+
+Indicate whether model `m1` is nested in model `m2`, i.e. whether
+`m1` can be obtained by constraining some parameters in `m2`.
+Both models must have been fitted on the same data. The mixed model must be
+fitted with maximum likelihood (i.e. **not** REML).
```

Comment:
Add a note here too saying that the nesting isn't strictly checked (and in the case of the non-formula-bearing GLMs, isn't really checked at all)?

Reply:
Actually, this should just be a method of our internal `_is_comparable` because `lrtest` doesn't allow models of different types anyway, and the documentation for `likelihoodratiotest` already discusses the lack of checking.